api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -339,7 +339,11 @@ public static boolean folderContains(File folder, String filename) {
 			return false;
 		}
 		
-		for (File f : folder.listFiles()) {
+        File[] fileList = folder.listFiles();
```

Comment:
Fix formatting. Run formatter phase in maven. fileList==null doesnt do anything here since folder.isDirectory is checked above and listFiles returns null only if the File is not a directory. It will return empty if the directory doesn't contain any files